.gitignore

```diff
@@ -57,3 +57,8 @@ docs/_build/
 # PyBuilder
 target/
 
+# PyCharm
+.idea/
```

Comment:
You can add this to your [global .gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore) so you don't have to add it to every repo, editor and environment specific gitignores should go there